.circleci/config.yml

```diff
@@ -509,7 +509,13 @@ jobs:
             SA_EMAIL="$(echo "$SA_KEY_JSON" | jq -r .client_email)"
             echo "$SA_KEY_JSON" | gcloud auth activate-service-account "$SA_EMAIL" --key-file=-
 
-            ../e2ev2/src/circle-build-ui.sh
+            export PR_NUM="$(echo "$CIRCLE_PULL_REQUEST" | perl -ne "/(\d+)\$/; print \$1")"
```

Comment:
Where does $CIRCLE_PULL_REQUEST come from?

Reply:
This, among many others, is set by Circle before starting the job. My preferred way to investigate the environment is to re-run a job with ssh, which is why the Playground job exists. Just run `env` in there to see everything.

Reply:
I like that. That would be a helpful way to look into this. Thanks for pointing it out.